src/configuration/services/mysql.md

```diff
@@ -146,11 +146,10 @@ configuration:
 
 ## Access your MariaDB service
 
-Assuming your MariaDB relationship is named `database`, you can access it by connecting
-to a host named `database.internal` using the MySQL command line client.
+Assuming your MariaDB relationship is named `database`, the host name and port number obtained from `PLATFORM_RELATIONSHIPS` would be `database.internal` and `3306`. Open an [SSH session](/development/ssh.md) and run the MySQL command line client.
 
 ```bash
-mysql -h database.internal -u user main
+mysql -A -h database.internal -P 3306 -u user main
```

Comment:
I don't really know why it's a bit slow when the client is connecting to our HA MariaDB Galeria cluster. The question is do we want our customer to enjoy fast startup anywhere versus auto-complete.